### DISCUSSION TITLE
Add conferencing options

### DISCUSSION THREAD

**Discussion by L1qu1d1c3** [twentyhq/twenty#5182] [Ideas]
It would be awesome if we could set up calls with leads/customers via Google Meet or MS Teams directly from the app. Also just general meetings with them

**Comment by BOHEUS** [comment]
It should be possible to do an app with such functionality once SDK is stable (most likely this quarter)

I checked and now it's possible to add the meeting via API  
https://developers.google.com/workspace/meet/api/guides/meeting-spaces
https://tomkarho.com/blog/creating-a-teams-meeting-using-graph-api

**Comment by BOHEUS** [comment]
Also, related to #7008
